package.json

```diff
@@ -1,20 +1,22 @@
 {
   "name": "wdio-jest",
-  "version": "1.4.2",
+  "version": "2.0.0",
   "description": "WebdriverIO boilerplate using WebdriverIO, Jest and Wiremock",
   "scripts": {
-    "test": "jest"
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest"
   },
   "author": "Erwin Heitzman",
   "license": "MIT",
   "dependencies": {
-    "@types/jest": "^26.0.24",
-    "@types/node-fetch": "^2.5.12",
-    "jest": "^27.0.6",
-    "node-fetch": "^2.6.1",
-    "ts-jest": "^27.0.4",
-    "typescript": "^4.3.5",
-    "wdio-wiremock-service": "^6.0.6",
-    "webdriverio": "^7.9.0"
+    "@jest/globals": "^29.6.2",
```

Comment:
This should not have to be manually installed as far as I am aware.

Reply:
This is the recommended approach according to ​​​https://jestjs.io/docs/getting-started#type-definitions as far as I understood.